src/EventListener/TemplateListener.php

```diff
@@ -97,7 +97,10 @@ private function addCategoriesToTemplate(FrontendTemplate $template, Module $mod
 
         $template->categories = $data;
         $template->categoriesList = $list;
-        $template->class = \implode(' ', \array_unique($cssClasses));
+
+        if ($cssClasses = \array_unique($cssClasses)) {
```

Comment:
Can you add the wrapping `count($cssClasses = ...) > 0` here please?

Reply:
Sure. Though it's technically not necessary, since an empty array evaluates to `false` as well.